src/Eshopworld.Tests.Core/Eshopworld.Tests.Core.csproj

```diff
@@ -23,7 +23,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Eshopworld.Tests.Core.Analyzers" Version="1.0.5" />
+    <PackageReference Include="Eshopworld.Tests.Core.Analyzers" Version="1.0.6" />
```

Comment:
Technically this means you should update the Tests.Core package as well. Unless that hasn't been deployed yet.

Reply:
It hasn't that's why I did not bump the version there. But I can if you guys deem it appropriate.

Reply:
No, that's ok. I just checked and both feeds are on -preview1